src/Features/Analytics.php

```diff
@@ -134,8 +134,8 @@ public function register_pages() {
 			array(
 				'id'     => 'woocommerce-analytics-customers',
 				'title'  => __( 'Customers', 'woocommerce-admin' ),
-				'parent' => 'woocommerce-analytics',
-				'path'   => '/analytics/customers',
+				'parent' => 'woocommerce',
+				'path'   => '/customers',
```

Comment:
Should this be broken out from the analytics "feature" now?

Reply:
Perhaps we could break it into its own feature... because it seems like there may be some child features of it too coming up.